### ISSUE TITLE
Need an EPUB title to show off DPUB ARIA features 

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/transitiontoepub#43]
Developers have asked us for a EPUB title they can use to demonstrate the speaking of DPUB ARIA features. The title should be of interest to many people and have most of the DPUB ARI attributes. This will help to promote the feature to screen reader users and to show the general public its effectiveness.

Once this is available, the screen readervelopers will want to make it available to their user base.

This raises the problem of having a generally interesting title that we have the rights to distribute.

**Comment by gregoriopellegrino** [comment]
Do you mean a "real" EPUB, not the test one we have now? That contains examples for all DPUB ARIA roles?

**Comment by GeorgeKerscher** [comment]
Yes, a real title that can be used to show people what it is like while reading an interesting title. The World Cultures and Geography comes to mind because we already have copyright release for that. We also have Front Page Inside Out, but that one is not as interesting.

**Comment by Rossolson** [comment]
I am working on a math textbook that has fallen out of copyright for Project Gutenberg. The notes are here as we continue to hash out a modern example of how to author a maths text. https://www.pgdp.net/wiki/User:Rossolson/Math#Research_and_Discussions and I would be happy to customize and revise the book to make it a high quality example of a larger work, not just a simple test case. (Though a subset of the book could be extracted for tests.)

Please let me know if this is of interest.

**Comment by Rossolson** [comment]
The EPUB file as it is built today is at https://ebookmaker.pglaf.org/cache/20250527194118/10000004-images-epub3.epub but it has NOT been enhanced to include any ARIA directives or other affordances.

My proposal is that people would review the work, and point out changes that should be made. I would make those changes and make the revision available. We would do this as often as necessary to get a high quality example of Accessible Math In EPUB.

As for the CC license, this is unnecessary, at least in the US. The book has passed out from copyright protection due to its age. Fortunately, math itself hasn't expired.
